### ISSUE TITLE
Configuration File not loading

### ISSUE THREAD

**Issue by shankar78be** [itgalaxy/webfont#2]
Hi,

When i run the below command the icon is created only the svg file specified in the cmd line. I have specified 2 files in the webfont.config.json. How to create icons for the 2 SVG files specified in the json file.

cmd> webfont "src/assets/images/action-sheet-check.eps.svg"  --dest "src/assets/icons2"  -s "src/assets/icons2/dfw.css" --config "webfont.config.json"

webfont.config.json
==============
{
  "files": [
    "src/assets/images/action-sheet-check.eps.svg",
    "src/assets/images/arrow-left-black-sm.eps.svg"
  ],
  "dest": "src/assets/icons2",
  "dest-css-template": "src/assets/icons2/dfw.css",
  "fontName": "my-font-name"
}

**Comment by evilebottnawi** [comment]
@shankar78be Thanks for reporting

**Comment by evilebottnawi** [comment]
@shankar78be you want create two fonts?

**Comment by shankar78be** [comment]
Yes, I want to create web fonts for the 2 images specified in the config file
"src/assets/images/action-sheet-check.eps.svg"
"src/assets/images/arrow-left-black-sm.eps.svg"

**Comment by evilebottnawi** [comment]
@shankar78be Poor understand, describe the problem in detail what you want to get and what you get is actually

**Comment by shankar78be** [comment]
I have configured the package.json to generate the webfonts. In the prebuild, the webfont were configured with the following parameters input, dest & destcsstemplate. It works fine, created webfonts for the SVG images located inside the images folder. 
"scripts": {
"prebuild": "webfont src/assets/images/*.svg  --dest src/assets/icons  -s src/assets/icons/dfw.css",
"start": "npm run server"
}


If I need to generate webfonts based on the configuration specified webfont.config.json, what needs to be done ?

webfont.config.json
------------------
{
 "files": [
   "src/assets/images/action-sheet.svg",
   "src/assets/images/arrow-left.svg"
],
"dest": "src/assets/icons",
"dest-css-template": "src/assets/icons/dfw.css",
"fontName": "webfont"
}


Need help on this.

**Comment by evilebottnawi** [comment]
@shankar78be Thank you for the detailed description, it is now clear, tomorrow I'll see what I can do, i see you want give an opportunity use `files` (glob) and `dest*` options in `webfont.config.json`. Now it is supported only for CLI.

**Comment by evilebottnawi** [comment]
@shankar78be you can use `files` for CLI or in config file, it can not be used in both, i fixed this throw error if you use CLI and config file contain `files`.

**Comment by jimmyandrade** [comment]
@shankar78be did you manage to solve what you needed? If so, I will be happy to close this issue. Otherwise, please tell me to see how I can help you.

**Comment by jimmyandrade** [comment]
Thanks for the detailed report — we're looking into this.

**Investigation:** the CLI always used positional arguments for `files`, so a config-only `files` array was ignored when no SVG paths were passed on the command line.

**Fix in progress:** https://github.com/itgalaxy/webfont/pull/696 — allows `webfont --config webfont.config.json -d out/` with `files` in the config, and errors clearly when both CLI paths and config `files` are set.

We'll keep this issue open until the fix is included in a published release on npm. It is targeted for the **next** milestone.

**Comment by jimmyandrade** [comment]
Update on documentation: upgrade steps for this fix will live in [MIGRATION.md](https://github.com/itgalaxy/webfont/blob/fix/issue-2-config-files/MIGRATION.md) (before/after across versions), not TROUBLESHOOTING.md. Workarounds on 12.0.0 and steps after the next npm release are documented there in PR https://github.com/itgalaxy/webfont/pull/696.

**Comment by jimmyandrade** [comment]
**Released on GitHub:** [v12.0.1](https://github.com/itgalaxy/webfont/releases/tag/v12.0.1) includes the fix for config `files` ([#696](https://github.com/itgalaxy/webfont/pull/696)).

**Upgrade steps** (once the package is on npm): see [MIGRATION.md § 12.0.1](https://github.com/itgalaxy/webfont/blob/master/MIGRATION.md#1201--cli-files-from-config-2).

```shell
npm install webfont@12.0.1
webfont --config webfont.config.json -d dist/icons
```

**npm publish:** CI publish was triggered via `workflow_dispatch` but failed with `EOTP` — the `NPM_TOKEN` secret needs to be an npm **Automation** or granular token with **bypass 2FA** for publish. We will retry after updating the token. Keeping this issue open until `12.0.1` is available on the npm registry.
